README.md

```diff
@@ -89,20 +91,50 @@ CROP is implemented using a well established software stack (given below) and ex
+This folder mainly exists to hold the `host.json` and `function.json` files for each function.
+* `models` has the code for the forecasting models, ARIMA and GES.
+* `media` has illustrations, logos, etc.
+* `utils` and `misc_scripts` have various utilities that don't fit the other categories.
+* `.secrets` has shell scripts for setting environment variables with secrets, such as database passwords, to facilitate running a local copy of the webapp for development use.
+The version-controlled files are merely templates, to be filled in with the actual values for the secrets.
+
+All of `tests`, `webapp`, `ingress_functions`, `utils`, and `models` import and use various parts of `core`. None of them communicate with each other.
+
+The repository root also has three Docker files:
+* `Dockerfile_ingress_functions` builds a container that holds all the functions in `ingress_functions`, in an environment to run them on an Azure function app.
+* `Dockerfile_webapp` builds the webapp, ready to be deployed as an Azure app service. It builds on `webappbase`.
+* `Dockerfile_webappbase` is a container that installs some of the dependencies needed by the webapp.
+It's separated from `webapp` to improve build times, and shouldn't need to be rebuilt except in rare circumstances.
+
+Note that even though the GES model also runs as an Azure function app, it has no Dockerfile. This is because it is deployed directly from the Github Action.
+
+Some of the subfolders have their own READMEs. See them for more details of each part.
 
 ### Development credits
 
 [Gentelella](https://github.com/afourmy/flask-gentelella) - a free to use (MIT license) Bootstrap admin template on which the webapp is built.
 
 ## Deployment
 
-#### Continuous Deployment
-
-We employ a continuous delivery toolchain with Docker on Travis CI which publishes Docker images automatically when PR is made to the *master* or *develop* branches.
+We employ a continuous delivery toolchain using Github Actions which deploys the latest version of CROP when a push or a PR is made to the `main` or `develop` branches.
+The Github Actions
+* Build a Docker container for the webapp, and push it to Docker Hub.
+* Build a Docker container for the Azure function apps that collect data into the database (ingress functions), and push it to Docker Hub.
+* Publish the Azure function app for running the forecasting models.
+* Run the CROP test suite.
 
-#### Manual
+The Azure services for the webapp and the ingress functions listen to updates on Docker Hub, and deploy the latest container once it has been pushed to the Hub.
 
-Steps of how to set up and deploy CROP locally or on Azure are documented [here](https://github.com/alan-turing-institute/CROP/wiki/1.-Set-up-and-deploy-CROP-locally-or-on-Azure).
+`main` and `develop` are deployed to production and testing versions of the platform, respectively. The former is relatively stable, the latter may be broken in the course of development.
```

Comment:
Maybe "The `main` and `develop` branches ..." ?

Reply:
Done